docs/Software/contributing.md

```diff
@@ -0,0 +1 @@
+../CONTRIBUTING.md
```

Comment:
I think this symlink should be ../../CONTRIBUTING.md, right ?

Reply:
You're right